### ISSUE TITLE
Can I set  the refresing icon not show when refresh is triggered by a swipe gesture

### ISSUE THREAD

**Issue by peterforme** [OrangeGangsters/SwipyRefreshLayout#67]
I don't want the refresing icon show when refresh is triggered by a swipe gesture , because I want to show a loading dialog in other way

**Comment by pihariev** [comment]
Yes, you can by using reflection. Look at the accepted answer: [TAP](https://stackoverflow.com/questions/45621101/swiperefreshlayout-without-progress-spinner)
